cookiecutter.json

```diff
@@ -3,8 +3,9 @@
   "project_slug": "{{ cookiecutter.project_name.lower().replace(' ', '_').replace('-', '_') }}",
   "project_url": "https://github.com/Deltares/{{cookiecutter.project_slug}}",
   "docs_url": "https://deltares.github.io/{{cookiecutter.project_slug}}",
-  "model_name": "PluginModel", 
-  "main_file_basename": "{{ cookiecutter.model_name.lower().replace('model','') }}", 
+  "model_classname": "PluginModel",
+  "model_type": "Model", 
```

Comment:
we can include this model_type part of the template but do note that the user can fill in anything they like here, so there will be no checking, and it's very possible to generate a broken template  if you make a typo here. If you're fine with that we can include this, but personally I'm hesitant.

Reply:
I think it helps to get a good start to start with the correct submodel class. Maybe I can include the list in the suggestions rather than just Model (Model / GridModel / MeshModel / LumpedModel)? Shame we can't do check with cookiecutter but in a way any typo in the cookiecutter argument needs to be checked so I still think it's nice to include

Reply:
there is a mechanic in coockiecutter to be able to choose from options so that would help but currently you can only choose one. There is a feature request to be able to choose multiple options, but that is for now unmerged and I don't know if/when that will land. I think for now this is fine, and we can always revisit it later when that feature is published.